contracts/VolOracle.sol

```diff
@@ -1,41 +1,64 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity >=0.8.4;
 
-import { Observation } from "@uniswap/v3-core/contracts/libraries/Oracle.sol";
 import { IUniswapV3Pool } from "@uniswap/v3-core/contracts/interfaces/IUniswapV3Pool.sol";
 
 contract VolOracle {
     // constructor
     // initializes a uniswap
 
+    struct VolObservation {
+        // the block timestamp of the observation
+        uint32 blockTimestamp;
+        // the tick accumulator, i.e. tick * time elapsed since the pool was first initialized
+        int56 tickCumulative;
+        // the tick square accumulator, i.e. tick * tick * time elapsed since the oracle was first initialized
+        uint112 tickSquareCumulative;
+        // whether or not the observation is initialized
+        bool initialized;
```

Comment:
we might not need this, since this was used because Observations array can grow in size over time, we want the array to be always of constant size